### ISSUE TITLE
Different javaw.exe instances will not accept different icons

### ISSUE THREAD

**Issue by Stewie410** [jonseppanen/Raindock#11]
Knowing full well that your code is based on the <name1>.exe comparing to <name1>.png, this is more of a question than a bug.

Minecraft (Java version), which is technically "javaw.exe", will only accept an icon as "javaw.png"--though Raindock is able to note that the instance is for Minecraft 1.x.x.  My question, is possible provide different icons for the same process, to be used for different instances of the same executable?

Ideally, it would be great to be able to either:
1.  `if [[ ! -f "$dir/$exeName.png" ]]; then icon=$name.instance.png; fi`
2.  `if [[ ! -f "$dir/$exeName.png" ]]; then icon=$instanceName.png; fi`
3.  `if [[ "$userOverride" == true ]]; then icon=$userIcon.png; fi`

_Apologies for the 'bash' style logic, its just the language I've been using lately._

Below, I've attached screenshots with my current configuration using the javaw.exe && Minecraft 1.10.2 instance as an example:

Dock w/ javaw.exe (minecraft) not pulling minecraft.png
![javaw_minecraft_on_dock](https://user-images.githubusercontent.com/24593067/35312471-98a6467a-0089-11e8-90ac-4819dd4b32f2.png)

%userprofile%\raindock\Default directory (current theme location)
![minecraft_png_in_dir](https://user-images.githubusercontent.com/24593067/35312512-d6cdde86-0089-11e8-89bd-32ca965fb4ce.png)

Task Manager showing javaw.exe instance "Minecraft 1.10.2"
![minecraft_task_manager](https://user-images.githubusercontent.com/24593067/35312536-f10932dc-0089-11e8-8fae-b6c915009b83.png)
----

If this is either 1) outside the scope of raindock, or 2) would otherwise break the core directives of raindock; then that's okay too.

**Comment by jonseppanen** [comment]
This is easily doable. Thanks so much for the detailed report! Will tackle this in the release after next.

**Comment by Stewie410** [comment]
Of course!  If I can provide any additional information (or examples), I'd be happy to.
As an aside, I get bug report tickets and other support tickets all day, and the most information I get is "HALP!"--I just can't be that person when submitting my own reports. :)
